haskell-tree-sitter.cabal

```diff
@@ -1,31 +1,30 @@
+cabal-version:       2.2
 name:                haskell-tree-sitter
 version:             0.1.0
 synopsis:            haskell tree-sitter bindings
 description:         Please see README.md
 homepage:            http://github.com/tree-sitter/haskell-tree-sitter#readme
-license:             BSD3
+license:             BSD-3-Clause
 license-file:        LICENSE
 author:              Rob Rix, Josh Vera, Tim Clem, Rick Winfrey, Max Brunsfeld, Ayman Nadeem, Patrick Thomson
 maintainer:          rob.rix@github.com
 copyright:           2015 GitHub
 category:            Web
 build-type:          Custom
--- extra-source-files:
-cabal-version:       >=1.10
 
 library
   hs-source-dirs:      src
-  build-depends:       base >= 4.7 && < 5
-                     , directory >= 1.3.0.2 && < 1.4
-                     , filepath >= 1.4.1.2 && < 1.5
-                     , split >= 0.2.3.2 && < 0.3
-                     , template-haskell >= 2.12.0.0 && < 2.15.0.0
-                     , unordered-containers
-                     , aeson
-                     , text
-                     , hedgehog
-                     , bytestring
-                     , fused-effects
+  build-depends:       base                  >= 4.7 && < 5
```

Comment:
Trying to be a better citizen about version bounds so that we can use cabal new-build and stack.